Project/core/src/main/java/View/screen/SetupScreen.java

```diff
@@ -37,7 +37,7 @@ private void setupUI() {
         float width = stage.getWidth();
         float height = stage.getHeight();
 
-        Label question = new Label("How many players?", application.skin, "narration");
+        Label question = new Label("How many users?", application.skin, "narration");
```

Comment:
I think "players" is still more intuitive for the users to read, since they aren't aware of our internal variable names. 
```suggestion
        Label question = new Label("How many players?", application.skin, "narration");
```